pkg/pmem-csi-operator/controller/deployment/controller_driver.go

```diff
@@ -377,6 +424,7 @@ var subObjectHandlers = map[string]redeployObject{
 	},
 	"metrics service": {
 		objType: reflect.TypeOf(&corev1.Service{}),
+		enabled: controllerEnabled,
```

Comment:
Node metrics service cant be enabled regardless of the controller?

Reply:
Node metrics are always enabled. There's nothing in the operator API to control that.